CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # TODO Darius: Find CMake minimum required version.
```

Comment:
@neatudarius is this line still relevant?

Reply:
If this is the new minimum required version,  we can remove the todo